src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -636,8 +643,8 @@ ov::npuw::LLMCompiledModel::LLMCompiledModel(const std::shared_ptr<ov::Model>& m
     if (optimize_v_tensors) {
         LOG_DEBUG("6. Check and apply opt layout");
         LOG_BLOCK();
-        if (optimize_value_tensors(kvcache_model)) {
-            NPUW_ASSERT(optimize_value_tensors(prefill_model));
+        if (ov::npuw::util::optimize_value_tensors(kvcache_model, false)) {
+            NPUW_ASSERT(ov::npuw::util::optimize_value_tensors(prefill_model, true));
```

Comment:
This code can be left as-is if we always tag the high-precision Add in SDPA (regardless if it is Prefill or KVcache)?